Content.Server/DeltaV/SpaceFerret/Systems/CanBackflipSystem.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Content.Server.Actions;
+using Content.Shared.DeltaV.SpaceFerret;
+
+namespace Content.Server.DeltaV.SpaceFerret.Systems;
+
+public sealed class CanBackflipSystem : EntitySystem
+{
+    [Dependency] private readonly ActionsSystem _actions = default!;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        SubscribeLocalEvent<CanBackflipComponent, MapInitEvent>(OnInit);
+        SubscribeLocalEvent<CanBackflipComponent, BackflipActionEvent>(OnBackflipAction);
+    }
+
+    private void OnInit(EntityUid uid, CanBackflipComponent component, MapInitEvent args)
+    {
+        _actions.AddAction(uid, ref component.BackflipActionEntity, component.BackflipAction, uid);
+    }
+
+    public void OnBackflipAction(EntityUid uid, CanBackflipComponent comp, BackflipActionEvent args)
+    {
+        RaiseNetworkEvent(new DoABackFlipEvent(GetNetEntity(uid), comp.ClappaSfx));
```

Comment:
as its stored on the component the sound shouldnt be networked at all, then dont need to validate it